FoxHound.Client/src/Posts/AddPost.tsx

```diff
@@ -10,26 +10,31 @@ import {
 } from "@material-ui/core";
 import Axios from "axios";
 import PostContentEditor from "./PostContentEditor";
+import { useParams } from "react-router-dom";
 
 const AddPost: React.FC = () => {
   const [title, setTitle] = useState<string>("");
   const [content, setContent] = useState<string>("");
   const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
   const [error, setError] = useState<string>("");
+  const { blogId } = useParams();
 
   const updateContent = (updatedContent: string) => {
     setContent(updatedContent);
   };
 
   const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
     event.preventDefault();
-
+    const tempId = blogId;
+    console.log(blogId);
     try {
       setIsSubmitting(true);
       const response = await Axios.post<number>(
         `${process.env.API_URL}/Post/Create`,
         {
-          blogId: 1,
+          //Look into getting the blog ID from the route
+
+          blogdId: tempId,
```

Comment:
- You can remove this comment.- 
- blogdId should be blogId on the left side
- remove the tempId const a nd just use blogId on the right side of this.
- further down in this file, a blogId is being defined and is set t o the response.data. this should actually be the postId. That's why blogId was saying it was used before it was defined. Changing that to postId is correct (since it's a postId anyway), this was an oversight on my part when i wrote this component.

Reply:
I updated the file per your suggesstions.  I noticed that
` const postId = response.data;` is defined, but never used. I looked elsewhere in the code for where blogID was referenced, but I didn't see anywhere else it was being used... We can talk more about this later if you would like.

Reply:
Eventually, I'd like to take the user the post itself after it's been created. We'll need the postId for that, but since that view post page isn't created yet, this postId is just kinda hanging out here for the time being.  Good observation though.